spec/requests/relationship_operations_spec.rb

```diff
@@ -282,7 +282,7 @@
 
     context 'unauthorized for remove_to_many_relationship' do
       before do
-        disallow_operation('remove_to_many_relationship', article, kind_of(Comment), :comments)
+        disallow_operation('remove_to_many_relationship', article, kind_of(Array), "comments")
```

Comment:
Same thing as in https://github.com/venuu/jsonapi-authorization/pull/52/files#r109101102, just that the assertion should be `disallow_operation` instead of `allow_operation`